### ISSUE TITLE
Sawdust to plank recipes do not use Ore Dictionary for sawdust, resulting in inability to utilize them if another mod inventory-unifies dustWood/pulpWood into a different item

### ISSUE THREAD

**Issue by Barhandar** [ReikaKalseki/Reika_Mods_Issues#1582]
JWIN, GregTech, (possibly) UniDict/UniQ will all result in this issue. Other mods that include sawdust items are GregTech, Forestry, Extra Trees, and Thermal Expansion, plus possibly more.

**Comment by ReikaKalseki** [comment]
Converting other mods' sawdust to planks could well introduce feedback exploits, as they may not all have the same equivalent value.

This is one reason why unifiers are a fundamentally bad idea.

**Comment by Barhandar** [comment]
>Converting other mods' sawdust to planks could well introduce feedback exploits, as they may not all have the same equivalent value.

It's the exact reverse, actually, letting other mods' items be used in RoC plank-from-sawdust-recipe would be incredibly lossy. **RotaryCraft is the feedback exploit**, as it has the 1:4 conversion while all other mods have _at most_ 1:2-and-only-from-logs (TE). GregTech is 1:1, Mekanism is 1:0.25 (1 sawdust for 1 log, to be more specific, 25% chance per plank, 100% per log), Forestry is 1:1 and also only lets logs be used, EIO is _kindasorta_ 1:1 as it has 90% chance but can use mill balls.  
_Nothing else produces as much sawdust from a plank as the Grinder_, and it works on GregTech's Wood Plank aka plateWood aka this-thing-made-from-**1**-dustWood-in-Compressor.  
Really easy to fix with access to modifying Grinder recipes, too, "if GT is detected, process plankWood into 4 dustSmallWood instead of 4 dustWood" negates the unifier problem entirely (and so does registering your sawdust as dustSmallWood only in the first place), "if the item has any OD entry but plankWood, and especially if it has plateWood, don't let it be used for sawdust in the Grinder" negates the specific loop.

>This is one reason why unifiers are a fundamentally bad idea.

No, this is why using same OD entries (_ones which have implicit function equivalence_ like dustX, at least) on items that perform differently is a fundamentally bad idea.

tl;dr: RotaryCraft generates more dustWood from plankWood than any other mod. All other mods produce 1 dustWood per 1 plankWood, only produce dustWood from logWood, or both. This means that RoC dustWood -> plankWood recipes are _lossy_ if using any other mod's dust, and that presence of any other means of turning dustWood into plankWood, due to them using 1:1 convention, enables feedback exploit in the Grinder.
